### PR TITLE
Add more information on installing caldera v4.

### PR DESCRIPTION
**Pull request by CDJellen**
## Description

Adjustments to the `Installing CALDERA` page in the readthedocs. Core changes are to the requirements (which exclude Python 3.10 until a `websockets` issue is resolved), to the section on launching caldera, and to the Docker overview.

## Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

All documentation changes were made to the markdown and previewed in Visual Studio Code to ensure proper links.  The new hyperlinks are worth reviewing, but I've matched the format for existing (and working) links as they appear elsewhere in `fieldmanual`.

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by CDJellen** [comment]
I'll request a merge after one approval, extra reviewers added only to expedite v4 documentation fixes.

File: sphinx-docs/Installing-CALDERA.md [hunk 1]
```diff
@@ -44,21 +65,37 @@ Start by cloning the CALDERA repository recursively, passing the desired version
 
 ```
 git clone https://github.com/mitre/caldera.git --recursive --branch x.x.x
+```
+
+Next, build the docker image, changing the image tag as desired.
+
+```
 cd caldera
+docker build . -t --build-arg WIN_BUILD=true caldera:server
```
Comment (uruwhy) [inline sphinx-docs/Installing-CALDERA.md:74]: should the `build-arg` go before the tag specification?

File: sphinx-docs/Installing-CALDERA.md [hunk 2]
```diff
@@ -1,34 +1,70 @@
...
+```sh
+git clone https://github.com/mitre/caldera.git
+--recursive --branch x.x.x
 ```
-git clone https://github.com/mitre/caldera.git --recursive --branch x.x.x
+
+To install version 4.0.0, one would execute:
+
+```sh
+git clone https://github.com/mitre/caldera.git
```
Comment (uruwhy) [inline sphinx-docs/Installing-CALDERA.md:49]: if folks copy-paste as-is, it'll be split into 2 lines in the terminal rather than 1

Comment (CDJellen) [reply sphinx-docs/Installing-CALDERA.md]: Good call, I've made the change,

**Review by uruwhy — APPROVED** [review]

**Pull request merged**